skorch/tests/test_net.py

```diff
@@ -2813,7 +3503,7 @@ def forward(self, x0, x1):
         class MyNet(NeuralNet):
             """Override train_step_single and validation_step"""
             def train_step_single(self, batch, **fit_params):
-                self.module_.train()
+                self._set_training(True)
```

Comment:
`self.module_` should still work when this PR is merged?

Given that `_set_training` is private API, I would not expect third-party developers to know about it.

Reply:
Right. So your point is that I should reverse the changes to this test, since it should not depend on using the private API? If so, I agree, I will reverse the changes.